js_modules/dagit/packages/core/src/overview/OverviewTimelineRoot.tsx

```diff
@@ -129,7 +138,9 @@ export const OverviewTimelineRoot = () => {
           </Box>
         </Box>
       </Box>
-      <RunTimeline loading={initialLoading} range={range} jobs={visibleJobs} />
+      <ErrorBoundary region="timeline" resetErrorOnChange={[range]}>
```

Comment:
`range` will change pretty often. Is that okay here? I could see it being an issue where an error is firing on every render, which would mean potentially re-triggering the error every time `range` changes, if I'm understanding this correctly.

Reply:
Ahhh good call, I will remove this. Without it present, you have to move away from the timeline or reload the page to try again, but i think that's fine.